documentation/practices.md

```diff
@@ -52,9 +39,28 @@ If you use another CMS, you must update the configurations to reflect the public
 
 **IBM Watson Content Hub (WCH) Integration**
 
-IBM Watson Content Hub is a cloud-based CMS with an embedded content delivery network that lets marketers update content faster. It also provides services with IBM Watson such as cognitive tagging, to help transform assets into a searchable library of content.
-Catalog images and site content images can be uploaded to WCH using the wchtools command-line utility. Uploaded assets are scanned using Watson's Visual Image recognition service, and tagged based on content to create a searchable library. After images are uploaded, the content delivery url must be provided in the reference storefront's configuration for both the skuImagesUrl and siteImagesUrl with the appropriate sku/file-name placeholders. For example, certain fields in the following examples are populated based on the tenant ID of the WCH account being used, and the directory structure of the assets uploaded.
+IBM Watson Content Hub is a cloud-based CMS with an embedded content delivery network that lets marketers update content faster. It also provides services with IBM Watson, such as cognitive tagging, to help transform assets into a searchable library of content.
+
+Catalog images and site content images can be uploaded to WCH using the wchtools command-line utility. Uploaded assets are scanned using Watson's Visual Image recognition service, and tagged based on content to create a searchable library. After the images are uploaded, the content delivery URL must be provided in the Reference Storefront's configuration for both the `skuImagesUrl` and `siteImagesUrl` with the appropriate sku/file-name placeholders. For example, certain fields in the following examples are populated based on the tenant ID of the WCH account being used, and the directory structure of the assets uploaded.
 `"skuImagesUrl": "https://my11.digitalexperience.ibm.com/<wch_tenant_identifier>/dxdam/<catalog_directory>/%sku%.jpeg",`
 `"siteImagesUrl": "https://my11.digitalexperience.ibm.com/<wch_tenant_identifier>/dxdam/<site_images>/%fileName%",`
 
+## Analytics Example
+
+**Google Analytics Integration**
+
+The React PWA Reference Storefront is pre-configured for integration with the Google Analytics enhanced e-commerce plugin. 
+
+### Route/Page Views
+The handlers for integrating with Google Analytics allow for routes, which are pre-configured in the `App.js` file, to be logged as pageviews in the sites real-time traffic data. Further configuration in Google Analytics will enable measurements for conversion rates based on the pages viewed in the web traffic. 
+`Analytics.js` contains the functions used for logging page views with ReactGA (React library for invoking Google Analytics functions), and functions for invoking the enhanced e-commerce capabilities.
```

Comment:
ReactGA (React library for invoking Google Analytics functions)
should be the other way around, abbreviation in bracket